test/scheduler/pickall_integration_test.go

```diff
@@ -0,0 +1,216 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package tests
+
+// This test suite features a number of test cases which cover the workflow of scheduling CRPs
+// of the PickAll placement type.
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	clusterv1beta1 "go.goms.io/fleet/apis/cluster/v1beta1"
+	placementv1beta1 "go.goms.io/fleet/apis/placement/v1beta1"
+)
+
+var _ = Describe("scheduling CRPs with no scheduling policy specified", Ordered, func() {
```

Comment:
This is still Ordered which is making it very difficult to reason for each test. Please make those independent. We don't need parallel but definitely not Ordered.

Reply:
Hi Ryan! This just means that all the test cases in this block run sequentially in **this** group; I marked this to avoid having to write a huge IT block

Reply:
Refactored per offline discussion.